fighthealthinsurance/views.py

```
@@ -622,6 +628,13 @@ def get(self, request, *args, **kwargs):
 
 class StripeWebhookView(View):
     def post(self, request):
+        try:
+            return self.do_post(request)
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)